.travis.yml

```diff
@@ -1,12 +1,12 @@
 language: go
 
 go:
-  - 1.8
-  - 1.9
-  - tip
+  - "1.9"
+  - "1.10"
```

Comment:
"1.10" requires quoting, because otherwise it is interpreted as 1.1. Quoted other values as well for consistency.